docs/submission/faq.md

```diff
@@ -5,3 +5,33 @@ platform_key: DOCS_FAQ
 ---
 
 ![Reminder Banner](/assets/submission/banner-reminder.svg)
+
+## Frequently Asked Questions
+
+1. How do I submit multiple values for a field? For example, I want to submit multiple symptoms for the `presenting_symptoms` field.
+
+   To include multiple values, separate values with a pipe delimiter '|' within your file. For example: `Anemia|Fatigue|Fever`.
+
+2. Is there any resource to refer to in order to follow staging guidelines?
+
+    Please refer to the guidelines provides by the [AJCC Cancer Staging](https://cancerstaging.org/Pages/default.aspx) website. Other helpful resources include the [CAnswer Forum](http://cancerbulletin.facs.org/forums/).
+
+3. The `tumour_staging_system` field does not contain the staging system I use. Can we request the staging system be added?
+
+    Yes, if the tumour staging system or tumour grading system you use is not included in the controlled terminology for `tumour_staging_system` or `tumour_grading_system`, please [contact us](https://platform.icgc-argo.org/contact) to request it be added.
+
+4. What assembly version do you use?
+
+    ICGC-ARGO uses GrCh38.
```

Comment:
- change ICGC-ARGO to ICGC ARGO.
- is it written like: GRCh38 Human Reference Genome (capitalized R)
https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26/